docroot/sites/all/modules/features/forms/templates/alumni-assoc-board-interest.tpl.php

```diff
@@ -0,0 +1,14 @@
+<?php print $salutation;?><br><br>
+
+Thank you for your interest in running for the ILR Alumni Association Board.<br><br>
+
+The next step is to complete and return the <a https://www.ilr.cornell.edu/sites/ilr.cornell.edu/files/ILRAANominationForm.docx">full nomination form</a>. The deadline and instructions are included on that form.<br>
+
```

Comment:
@carolineHecht what do you think about creating a vanity redirect for this url, in case it changes in the future?